camus-shopify/script/update_schedule

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+import os
+import sys
+import json
+import argparse
+import subprocess
+import logging as lg
+from os.path import basename, exists
+
+from azkaban import Job, Project
+from azkaban.remote import Session
+
+
+formatter = lg.Formatter('%(asctime)s %(levelname)s %(name)s %(message)s', '%d/%m/%y %H:%M:%S')
+ch = lg.StreamHandler(sys.stdout)
+ch.setFormatter(formatter)
+logger = lg.getLogger('azkaban_submit_job')
+logger.setLevel(lg.INFO)
+logger.addHandler(ch)
+
+def start_session(url, username, password):
+    session = Session(url)
+    session.user = username
+    session.password = password
+    session.id = None
+
+    return session
+
+
+def main():
+    repo_root = os.getcwd()
+    camus_jar = 'camus-shopify-0.1.0-shopify1.jar'
+    azkaban_auth = json.load(open(os.path.join(repo_root, 'camus-shopify', 'azkaban.json')))
```

Comment:
We never change camus-shopify version so I decided to keep it hardcoded cause I didn't find the benefit of adding that code.